### ISSUE TITLE
Physical Coin Sent To hltbb

### ISSUE THREAD

**Issue by privatereal** [bitshares/bitshares-ui#2720]
Hello Bitshares,

In order to understand the issue, please refer to screen grab attached here. Link is below. 

On February 24th, I attempted to send .5 worth of BTC off the exchange. In doing so, I errantly pressed the "Propose" button, thinking it was required to make this transaction. In doing so openlegder-dex created proposals for me, which I accepted not knowing what I was doing. A whole series of proposals continued to ensue, and I thought this was all required to get the money off the exchange. Only afterwards did I realize "Propose" should have never been pressed by me, and that I somehow "updated" my account in doing so. About 2 hours later all my physical coin was transferred to an account I am unfamiliar with: htlbb! I have no idea who this is or why my coins were transferred to this account, but I think it had something to do with the "Propose" button I pressed in making the original transaction. 

Is there any way to get my physical coin back? Has anyone seen this happen before to an account? It seems I was duped and committed to something unknowingly. Please review all these transactions happening on 02-24-19 to better understand issue. 

More recently, on 05-02-19 cryptobridge-upgrade created new proposals for me and charged me a 5.074BTS fee for those proposals. I'm not sure why that happened. Anyway my account has no more coin due to what happened on 02-24-19 and I'm wondering what I can do to get it back and what happened exactly on that day, if anyone can help me look into it. 

I wrote to Hackthedex initially, not know what I was doing and being kind of overly emotional. Now that I'm calm, I'm just trying to learn about this thing and what happened on that day.

![Screen Shot 2019-05-27 at 1 39 56 PM](https://user-images.githubusercontent.com/51094747/58435462-44d08500-8086-11e9-84ff-3663182990db.png)

Thanks for your help! Owner of PRVTRL

**Comment by startailcoon** [comment]
I'm sorry to say that the account `openlegder-dex` (note that it's a misspelling of OpenLedger DEX) has sent a proposal to change your account authority. By accepting this proposal, you gave another user access to making transactions on your behalf. The proposer then moved your coins to another account.

On 2019-02-24 14:08, a proposal was sent to you for updating your account data.
On 2019-02-24 14:14, you accepted an update of your account based on the proposal
On 2019-05-02 21:38, the same thing is happening by another scam account, trying to gain access to your account. The fee here isn't paid the sender of the proposal, not you, this is misleading. 

By going to your account and looking at your account permissions (Right Menu -> Permissions), you can see that your account has `openlegder-dex` added to its active permissions page.

Please read related topics 
- [Reminder: Be Cautious About Accepting Proposed Transactions](https://dex.openledger.io/news/reminder-proposals/)
- [Clearly render proposal contents](https://github.com/bitshares/bitshares-ui/issues/2499)
- [BSIP: Proposals Scam Prevention](https://github.com/bitshares/bsips/issues/154)

**Comment by privatereal** [comment]
Thanks for that. Hugely appreciated and glad I could help out in securing the site better by providing my example. I got the hit up for a proposal immediately after initiating a withdrawal off the exchange through openledger-dex. The proposal came within seconds of sending coin off the exchange by the hardly indistinguishable account "openlegder", as you know, and the rest is history. Pretty good scam if you ask me.  

I started going through proposed solutions to this "Proposal" problem which you provided and will continue to read. I appreciate you turning me on to them. I for sure think that "Proposals" should be made clear as to their intent in clear English when made, so the lay user knows what the consequences of accepting a proposal are. I also think warning signs should be posted on the "Propose" button to make clear the risks involved with pressing it and agreeing to a "proposal". I LOVE the idea of not making the "Propose" button available to every user unless the user pays for access to it and reads and agrees to terms of what it's used for. I also think the Bitshares community allow for 2 chances to say yes to something, to confirm the user absolutely knows what they're agreeing to and have read the proposal and are ready to commit. I think good actors, providing a service like openledger-dex, and other good third parties should be highlighted in distinct color to be trusted by the community. These would be colors that no other user gets. 

Anyway, I will keep reading what I can and lend any help I can in this matter to the community, because I loved the idea of a DEX like Bitshares to keep my assets SECURE! I'm just glad this loss didn't happen to my main account. 

I will do as you say and remove permissions for the scam account. Once that is done I assume I can use my account again without fear of anyone taking my funds, correct?

Thanks, PRVTRL
